<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BidistributionModelingDialog</class>
 <widget class="QDialog" name="BidistributionModelingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widDistrVar1" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblDistrVar1">
        <property name="text">
         <string>1) Select distribution for</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widDistrVar2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblDistrVar2">
        <property name="text">
         <string>2) Select distribution for</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="btnParameters">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/setting16</normaloff>:/icons/setting16</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>3) Review parameters and run:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnSave">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/save16</normaloff>:/icons/save16</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QPushButton" name="btnPlot">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/plot16</normaloff>:/icons/plot16</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>5) Save bidistribution:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>4) Plot result:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>(optional) Save new distribution for X variable:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>(optional) Save new distribution for Y variable:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnSaveXDistr">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/save16</normaloff>:/icons/save16</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btnSaveYDistr">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/save16</normaloff>:/icons/save16</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDismiss">
       <property name="text">
        <string>Dismiss</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnDismiss</sender>
   <signal>clicked()</signal>
   <receiver>BidistributionModelingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnParameters</sender>
   <signal>clicked()</signal>
   <receiver>BidistributionModelingDialog</receiver>
   <slot>onParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPlot</sender>
   <signal>clicked()</signal>
   <receiver>BidistributionModelingDialog</receiver>
   <slot>onPlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>BidistributionModelingDialog</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSaveXDistr</sender>
   <signal>clicked()</signal>
   <receiver>BidistributionModelingDialog</receiver>
   <slot>onSaveXDistr()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSaveYDistr</sender>
   <signal>clicked()</signal>
   <receiver>BidistributionModelingDialog</receiver>
   <slot>onSaveYDistr()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onParameters()</slot>
  <slot>onScatsmthCompletion()</slot>
  <slot>onPlot()</slot>
  <slot>onSave()</slot>
  <slot>onSaveXDistr()</slot>
  <slot>onSaveYDistr()</slot>
 </slots>
</ui>
